catalog/qod/samplecode_qod.md

```diff
@@ -481,13 +480,49 @@ app.listen(port, () => {
     console.log(`QoD authorization callback URL is running`);
 })
 ```
+```python Sample SDK for Python
+from flask import Flask, request, jsonify
+from aggregator_opengateway_sdk import ClientCredentials, QoDMobile, QoSProfiles
+
+credentials = ClientCredentials(
+    clientid='my-app-id',
+    clientsecret='my-app-secret'
+)
+
+app = Flask(__name__)
+
+@app.route('/qod-auth-callback', methods=['GET'])
+def auth_callback():
+    code = request.args.get('code', '')
+    qod_client = QoDMobile(client=credentials, auth_code=code)
+
+if __name__ == '__main__':
+    app.run()
+```
 
 #### API Usage
 
 Once we have instantiated the SDK class with the authorization code received from the frontend, we can just call the class service function to effectively create a QoD session for the frontend device, by getting its IP address from the HTTP request header.
 
 For that, still in the code of the auth callback URL endpoint listener, following to the QoDMobile class instantiation, follow the samples below:
 
+```node Sandbox SDK for Node.js
+
+    const result = await qodClient.sessions(
+        300,                                 // duration
+        undefined,                           // externalId
+        phoneNumber,                         // msisdn: The mobile number
+        undefined,                           // ipv4Addr
+        undefined,                           // ipv6Addr
+        undefined,                           // uePort
+        undefined,
+        'QOS_E',                             // QOS_E - Qualifier for enhanced communication profile
+        'http://my-server/notification-uri', //notificationUri: The URI for notifications
+        'my-notification-auth-token'         // notificationAuthToken
```

Comment:
I am afraid the answer is yes, but doesn't this method allow just not passing undefined parameters?